### PR TITLE
OSDOCS-4602: Observing network traffic using traffic flow

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.12

Issue: https://issues.redhat.com/browse/OSDOCS-4602

Link to docs preview: https://53366--docspreview.netlify.app/openshift-enterprise/latest/networking/network_observability/observing-network-traffic.html#using-traffic-flows

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by snarayan-redhat** [comment]
reopening the PR to generate preview

File: modules/network-observability-trafficflow.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+= Using Traffic flows
+Traffic flow provides the data of the network flows and the amount of traffic. As an administrator, you can monitor the amount of traffic across the application by using the traffic flow table.
+
+== Navigating to Traffic flows
+As an administrator, you can navigate to topology view by using the following steps:
```
[the remark this answers is not in the document]
Comment (snarayan-redhat) [reply modules/network-observability-trafficflow.adoc:5]: my bad! thanks for pointing that out.

File: modules/network-observability-trafficflow.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
+= Using Traffic flows
```
Comment (snarayan-redhat) [inline modules/network-observability-trafficflow.adoc:1]: Add id attribute

File: modules/network-observability-trafficflow.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Traffic flow provides the data of the network flows and the amount of traffic. As an administrator, you can monitor the amount of traffic across the application by using the traffic flow table.
+
+== Navigating to Traffic flows
+As an administrator, you can navigate to topology view by using the following steps:
+
+. On the left navigation panel, click *Observe* → *Network Traffic*.
+. In the *Network Traffic* page, click the *Traffic flows* tab.
+
+== Configuring advanced options
+You can customize and export the view by using advanced options. To access the advanced options, click *Show advanced options* on the right-top corner.
```
Comment (snarayan-redhat) [inline modules/network-observability-trafficflow.adoc:11]:
```suggestion
You can customize and export the view by using advanced options. To access the advanced options, click *Show advanced options*.
```

File: modules/network-observability-trafficflow.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
+= Using Traffic flows
+Traffic flow provides the data of the network flows and the amount of traffic. As an administrator, you can monitor the amount of traffic across the application by using the traffic flow table.
+
+== Navigating to Traffic flows
```
Comment (snarayan-redhat) [inline modules/network-observability-trafficflow.adoc:4]:
```suggestion
== Working with Traffic flows
```

**Comment by Amoghrd** [comment]
LGTM

**Review by jpinsonneau — APPROVED** [review]

**Comment by snarayan-redhat** [comment]
Closing this PR since pushing all the changes for this page in bulk in mentioned PR due to dependencies.

**Pull request closed without merging**